_layouts/home.html

```diff
@@ -1,37 +1,13 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    {% include head.html %}
-  </head>
-  <body itemscope itemtype="http://schema.org/WebPage">
-    <meta itemscope itemprop="url" content="{{ site.url }}{{ page.url }}">
-    {% if page.title %}
-      <meta itemscope itemprop="name" content="{{ page.title }}">
-    {% endif %}
+{% include head.html %}
 
-    <div class="homeheader">
-      <div class="homelogo">
-        <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-           viewBox="0 0 144 144" enable-background="new 0 0 144 144" xml:space="preserve">
-        <g>
-          <path fill="#E6E6E6" d="M72,60.601c-6.295,0-11.399,5.103-11.399,11.399c0,5.487,3.878,10.066,9.042,11.152v60.789
-            C70.427,143.966,71.21,144,72,144s1.573-0.034,2.357-0.06V83.152c5.164-1.086,9.042-5.665,9.042-11.152
-            C83.399,65.705,78.295,60.601,72,60.601z"/>
-          <path fill="#E6E6E6" d="M144,72c0-39.765-32.235-72-72-72S0,32.235,0,72c0,37.374,28.478,68.091,64.918,71.65l0,0l0.001-0.014h0
-            c-0.18-29.263-17.819-51.864-25.137-59.991v-0.002c0,0-3.006-3.604-1.303-7.924l13.328-33.805c0,0,2.466-6.249,7.945-6.249H72
-            h12.248c5.479,0,7.945,6.249,7.945,6.249l13.328,33.805c1.703,4.32-1.303,7.924-1.303,7.924v0.002
-            c-7.318,8.127-24.958,30.729-25.137,59.991l0.001,0.014C115.522,140.091,144,109.374,144,72z"/>
-        </g>
-        </svg>
-      </div>
-      <h1>Design<br/>Open</h1>
-    </div>
+<div class="homeheader">
```

Comment:
best practice class naming convention is `-` i.e. `home-header` or `home--header` and `home--logo` (this is just an aside)